### PR TITLE
Source Format Enforcement

### PR DESCRIPTION
**Pull request by rousskov**
This change is a reference point for automated CONTRIBUTORS updates.

### CODE REVIEW

File: src/tests/testCacheManager.cc [hunk 1]
```diff
@@ -54,7 +54,7 @@ CacheManagerInternals::testValidUrl(const AnyP::Uri &url)
         (void)ParseUrl(url);
     } catch (...) {
         std::cerr << "\nFAIL: " << url <<
-            Debug::Extra << "error: " << CurrentException << "\n";
```
Comment (kinkie) [inline src/tests/testCacheManager.cc:57]: Doesn't seem related to the PR

File: src/tests/testCacheManager.cc [hunk 2]
```diff
@@ -65,7 +65,7 @@ CacheManagerInternals::testInvalidUrl(const AnyP::Uri &url, const char *const pr
     try {
         (void)ParseUrl(url);
         std::cerr << "\nFAIL: " << url <<
-            Debug::Extra << "error: should be rejected due to '" << problem << "'\n";
```
Comment (kinkie) [inline src/tests/testCacheManager.cc:68]: Doesn't seem related to the PR

**Review by kinkie — CHANGES_REQUESTED** [review]
Seems there's spurious changes

File: src/tests/testCacheManager.cc (hunk 1 above)
Comment (rousskov) [reply src/tests/testCacheManager.cc:57]: This "Source Format Enforcement" PR (as all "Source Format Enforcement" PRs?) is the result of executing our `./scripts/source-maintenance.sh` script. That script may format sources and update CONTRIBUTORS. Both adjustment happened in this PR. Both are in-scope. Does this summary address your concerns?

**Review by kinkie — APPROVED** [review]
Aha yes. It's the first comment that threw me off

**Comment by rousskov** [comment]
> It's the first comment that threw me off

If you recall, that or a similar comment is required to advance the "manually audited version" of the CONTRIBUTORS file. See collectAuthors() description in scripts/source-maintenance.sh or `git grep -10 vettedCommitPhraseRegex`.
